source/index.html.md

```diff
@@ -49,3 +49,27 @@ These SDKs are wrappers around HTTP calls to the corresponding Skynet endpoints.
 You can view code examples in the dark area to the right. Switch to the
 programming language of your choice by clicking the tabs in the top right.
 </aside>
+
+## A Note About Language Differences
+
+Though we tried to keep the SDKs as similar to each other as possible, differences between the languages -- the idiosyncracies and best practices of each -- resulted in differences between the SDKs. We've noted them where necessary.
```

Comment:
This line supports the idea that all those differences are at the example level. At least that is where I would assume they are if I was reading through the docs for the first time.

Reply:
Can you explain what you mean? I tried to address this, let me know if it looks okay